src/routes/JobSummary/components/RecordsTable.js

```diff
@@ -18,6 +18,7 @@ import {
   Button,
   NoValue,
 } from '@folio/stripes/components';
+import { pagingTypes } from '@folio/stripes-components/lib/MultiColumnList';
```

Comment:
Import this from `stripes/components` as `MCLPagingTypes`, please
Component imports should happen through the @folio/stripes/${repo} namespace rather than directly via the repo's private path due to UIDATIMP-927